src/prms/Main_prms.f90

```diff
@@ -84,7 +84,7 @@ program prms6
 
 
   ! Initialize the simulation object
-  call model_simulation%init(Control_data)
+call model_simulation%init(Control_data)
```

Comment:
This looks like only an indentation change. Maybe revert it?

Reply:
I agree. The indent was probably just a typo - doesn't exist in the main 6.0.0_dev branch.

Reply:
This should be fixed.